.circleci/config.yml

```diff
@@ -129,6 +129,25 @@ jobs:
       - run:
           name: Build and Publish Docker Image to Docker Hub
           command: ./build_and_upload_image.sh insta_pics-downloader/Dockerfile codesmag/insta_pics_downloader
+  
+  grpc-server:
+    executor: job-runner
+    steps:
+      - checkout
+      - setup_remote_docker
+      - run:
+          name: Build and Publish Docker Image to Docker Hub
+          command: ./build_and_upload_image.sh api/grpcserver/Dockerfile codesmag/grpc-server
+  
+  envoy-proxy:
+    executor: job-runner
+    steps:
+      - checkout
+      - setup_remote_docker
+      - run:
+          name: Build and Publish Docker Image to Docker Hub
+          command: ./build_and_upload_image.sh api/envoy-proxy/Dockerfile codesmag/envoy-proxy
+
```

Comment:
Looks good 👍 
You are just missing the entry in the workflow at the end of the file